extensions/blocks/button/index.js

```diff
@@ -22,7 +22,7 @@ export const settings = {
 	supports: {
 		html: false,
 		inserter: false,
-		align: [ 'left', 'right' ],
+		align: [ 'left', 'center', 'right'  ],
```

Comment:
This is causing the tests to fail I'm afraid:

> extensions/blocks/button/index.js
> 25:37  error  Multiple spaces found before ']'  no-multi-spaces

```suggestion
		align: [ 'left', 'center', 'right' ],
```

Reply:
Doh! Fixed.